packages/ddex/webapp/client/src/components/Collection/Collection.tsx

```diff
@@ -33,41 +146,61 @@ export const Collection = ({ collection }: { collection: CollectionT }) => {
   let data: any, error, isLoading
   switch (collection) {
     case 'uploads':
-      ;({ data, error, isLoading } = trpc.uploads.listCollection.useQuery({
-        nextId,
-        prevId
-      }))
+      ;({ data, error, isLoading } = trpc.uploads.listCollection.useQuery(
```

Comment:
i missed flagging when this was added, but the react gods get mad when hooks are inside conditionals like this. this pr isn't making it any worse, so not a blocker but i'd change it to run every hook but pass the `enabled` flag like:

```typescript
const { data, error, isLoading } = trpc.uploads.listCollection.useQuery(
	{
		nextId,
		prevId
	},
	{
		refetchInterval: 10_000 // Refetch 10 seconds,
		enabled: collection === 'uploads'
	}
)
```

Reply:
ah yes. usually it throws an error about this but for some reason this flew under the radar

Reply:
fixed in https://github.com/AudiusProject/audius-protocol/pull/7506/commits/4510f79efca1b495660612203f9f04f0b2ec0093

it wasn't throwing an error because technically the switch logic as written (despite switch being a conditional operation) doesn't directly violate the rules of hooks... based on the case, each useQuery hook is called at the top level of its respective case block and gets executed on every render. each render path calls its respective hook without conditionally skipping it.

that being said. i cleaned it up to look more like the conventional pattern with a top level hook.